lib/elements/steel_reinforcement/rebar_hook.rb

```diff
@@ -0,0 +1,28 @@
+require 'db/base'
+require 'elements/steel_reinforcement/base'
+
+module Elements
+  module SteelReinforcement
+    class RebarHook < Base
+      attr_reader :angle, :length
+
+      def use_angle(angle)
+        @angle = angle
+        @length = calculate_length
```

Comment:
we are going to remove the `calculate_length`, but if we left it, I think we shouldn't have stored this as attribute, but as method instead, a method called `length` that does what `calculate_length` does

Reply:
yup yup

Reply:
yup yup